main.go

```diff
@@ -90,3 +106,8 @@ func main() {
 	// Print the beacon log!
 	printLog(beaconLogData)
 }
+
+type beaconConfig struct {
```

Comment:
Would you mind moving this type to the `types` section towards the top of the file?